hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,157 @@
+- Improved: Formatting & spacing of messages in chat ({{<issue 4026 >}})
+- Improved: Audio wizard (appearance & explanations) ({{<issue 4100 >}})
+- Improved: Echo cancellation settings ({{<issue 4113 >}}, {{<issue 4174 >}})
+- Improved: Users can now always choose to receive update-notifications for the client ({{<issue 4138 >}}, {{<issue 4182 >}})
+- Improved: Echo cancellation is now enabled by default on all platforms but MacOS ({{<issue 4214 >}})
+- Improved: Accessibility ({{<issue 4211 >}}, {{<issue 4312 >}})
+- Improved: Some settings pages ({{<issue 4240 >}}, {{<issue 4243 >}})
+- Improved: CoreAudio implementation ({{<issue 4254 >}})
+- Improved: The user is notified if a change (e.g. local mute) could not be saved permanently ({{<issue 4301 >}})
+- Improved: Server Browser UI ({{<issue 4291 >}})
+- Improved: Clear selection in shortcut settings after having removed shortcut to avoid accidental subsequent removal ({{<issue 4358 >}})
+- Improved: Noise cancelling & associated UI ({{<issue 4212 >}})
+- Improved: Use zeroconf instead of Bonjour on Windows if available ({{<issue 4494 >}})
+- Improved: Tooltips for shortcut settings ({{<issue 4543 >}})
+- Updated: Opus to v1.3.1 ({{<issue 3813 >}})
+- Changed: Text-to-speech is now disabled by default ({{<issue 4627 >}})
+- Removed: DirectSound support ({{<issue 3828 >}})
+- Removed: Qt4 support ({{<issue 3602 >}})
+- Removed: CELT 0.11.0 support ({{<issue 2045 >}})
+
+### Server
+
+- Added: Ability to log ACL and group changes (helpful for debugging) ({{<issue 4017 >}})
+- Added: Option to not ban connections from an IP that managed to connect succcessfully ({{<issue 4087 >}})
+- Added: Dedicated ACL for resetting comments/avatars ({{<issue 4196 >}})
+- Added: Config option which allows the server to remeber a user's channel for limited time only ({{<issue 4147 >}})
+- Added: Ability to load welcome message from file ({{<issue 4344 >}})
+- Fixed: Use a temporary keychain on macOS in order to avoid permission issues ({{<issue 4345 >}})
+- Fixed: Undefined behavior when logging SSL error ({{<issue 4452 >}})
+- Improved: Also log a client's OS ({{<issue 4035 >}})
+- Improved: CPU utilization by using TCP_NODELAY mode by default ({{<issue 4054 >}})
+- Improved: Use zeroconf instead of Bonjour on Windows if available ({{<issue 4494 >}})
+
+### Installer
+
+- Improved: Chinese translations ({{<issue 3807 >}}, {{<issue 3613 >}})
+- Improved: Uninstall no longer deletes murmur.ini file ({{<issue 4096 >}})
+- Complete refactoring ({{<issue 4574 >}})
+
+
+### Positional audio plugins
+
+- Added: General Source Engine support ({{<issue 3771 >}})
+- Added: Among Us ({{<issue 4571 >}})
+- Updated: GTA V ({{<issue 4059 >}})
+- Updated: Manual plugin's UI now shows a speaker's location ({{<issue 4352 >}})
+
+
+### Overall
+
+- The project has been migrated to be built with cmake instead of qmake ({{<issue 4252 >}})
+
+
+## Known issues
```

Comment:
This should maybe be mentioned somewhere above.

Reply:
Hm I don't think this is important enough to be listed before the changelog, let alone before any of the other content :thinking:

Reply:
I think this is the correct place. The headline is on the same level as the Changelog, and follows it. This is also standard practice (elsewhere) from my experiecne.